docs/gravitino-server-config.md

```diff
@@ -113,16 +113,23 @@ To leverage the event listener, you must implement the `EventListenerPlugin` int
 
 #### Event
 
-Gravitino triggers a specific event upon the completion of the operation, with varying events being generated for different operations.
+Gravitino triggers a pre-event before the operation, a post-event after the completion of the operation and a failure event after the operation failed.
```

Comment:
You can have two tables, one for post-event, one for pre-event, and clearly add a heading to each table.